runtime/tests/tests.rs

```diff
@@ -120,6 +121,85 @@ fn transfer_fee<E: Encode>(extrinsic: &E, fee_multiplier: Fixed64, runtime_call:
 	base_fee + fee_multiplier.saturated_multiply_accumulate(length_fee + weight_fee)
 }
 
+#[test]
+fn staking_genesis_config_works() {
+	let balance_amount = 10_000 * TransactionBaseFee::get();
+	ExtBuilder::default()
+		.initial_balance(balance_amount)
```

Comment:
We could add `.stash(balance_amount / 5)` here if we use the ExtBuilder setter pattern